api/client/samples/crop_models/sao_paulo_sugar_content/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3628 @@
+{
```

Comment:
<p>explain that these two series are:  semi-monthly and annual</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>